client/trino-jdbc/pom.xml

```diff
@@ -12,7 +12,6 @@
     <artifactId>trino-jdbc</artifactId>
 
     <properties>
-        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
```

Comment:
Per airbase docs

> For a multi-module project, all other sub-modules must have this explicitly set to the root directory and therefore the following code
> 
> ```xml
> <properties>
>   <air.main.basedir>${project.parent.basedir}</air.main.basedir>
> </properties>
> ```
> must be added to each pom.

https://github.com/airlift/airbase/blob/master/README.md#airmainbasedir

so when using airbase root pom, we are expected to define the `air.main.basedir` property correctly, and this should not be removed

howeever, airbase seems to make no direct use of `air.main.basedir` property, so we may perhaps simply remove it from airbase (remove from airbase pom and readme). this would be a prep step, if we're pedantic.

Reply:
> this would be a prep step, if we're pedantic.

Oh, but we are!